convert_fendl.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python3
+
+import os
+from collections import defaultdict
+import sys
+import tarfile
+import zipfile
+import glob
+import argparse
+from string import digits
+from urllib.request import urlopen, Request
+
+import openmc.data
+
+
+description = """
+Download FENDL 3.1d or FENDL 3.1c ACE data from the IAEA and convert it to a HDF5 library for 
+use with OpenMC.
+
+"""
+
+
+class CustomFormatter(argparse.ArgumentDefaultsHelpFormatter,
+                      argparse.RawDescriptionHelpFormatter):
+    pass
+
+parser = argparse.ArgumentParser(
+    description=description,
+    formatter_class=CustomFormatter
+)
+parser.add_argument('-b', '--batch', action='store_true',
+                    help='supresses standard in')
+parser.add_argument('-d', '--destination', default=None,
+                    help='Directory to create new library in')
+parser.add_argument('--libver', choices=['earliest', 'latest'],
+                    default='latest', help="Output HDF5 versioning. Use "
```

Comment:
Does `earliest` not work for this library? The downside to using `latest` is that it requires the user to compile/run OpenMC against HDF5 1.10, i.e. if they have HDF5 1.8 it won't work.

Reply:
Changed to `earliest`